Makefile

```
@@ -58,10 +58,14 @@ GOPATH ?= ${HOME}/go
 # Environment variables set when running the Go compiler.
 GOOS ?= $(shell ${GO} env GOOS)
 GOARCH ?= $(shell ${GO} env GOARCH)
+CGO_ENABLED ?= 0
 GO_BUILD_ENVVARS = \
 	GOOS=$(GOOS) \
 	GOARCH=$(GOARCH) \
-	CGO_ENABLED=0 \
+	CGO_ENABLED=$(CGO_ENABLED)
+
+# Extra build flags passed to the go compiler.
+GO_BUILD_FLAGS = 
```

I believe you want `?=` here - to tell the makefile "if this env var is already set, don't set it here"... the way this is now, this env var will always be empty even if the one invoking the make file exported that env var to this process.
```suggestion
GO_BUILD_FLAGS ?= 
```

---

Yep!